parser-typechecker/src/Unison/Codebase/Editor/RemoteRepo.hs

```diff
@@ -80,28 +68,3 @@ isPublic ReadShareLooseCode {path} =
   case path of
     (segment Path.:< _) -> segment == NameSegment.publicLooseCodeSegment
     _ -> False
-
-data WriteRemoteNamespace a
```

Comment:
This type is replaced with just the ProjectBranch variant, removing the ability to write to loose-code on Share (which is already disabled in trunk UCM anyways)